components/ActionSheet/BUILD

```diff
@@ -193,9 +193,25 @@ mdc_objc_library(
     ],
 )
 
+mdc_objc_library(
+    name = "unit_test_helpers",
```

Comment:
Please name this "TestingSupport" so that our `.kokoro` rewrite scripts will work for it as well.